src/config.c

```diff
@@ -862,33 +857,26 @@ void configGetCommand(client *c) {
     int matches = 0;
     int i;
 
-    for (standardConfig *config = configs; config->name != NULL; config++) {
-        int matched_conf = 0;
-        int matched_alias = 0;
-        for (i = 0; i < c->argc - 2 && (!matched_conf || !matched_alias); i++) {
+    dictIterator *di = dictGetIterator(configs);
+    dictEntry *de;
+    while ((de = dictNext(di)) != NULL) {
+        standardConfig *config = dictGetVal(de);
+        for (i = 0; i < c->argc - 2; i++) {
```

Comment:
maybe we wanna swap the order of the loops, and check if the argument doesn't have a wildcard in it, use dict lookup rather than iterator?

Reply:
I did try this out. It's a good bit more code, but did resolve the performance delta.